.stickler.yml

```diff
@@ -0,0 +1,9 @@
+linters:
+  csslint: {  }
+  stylelint: 
+    config: 'stylelint.config.js'
+files:
+  ignore:
+    - 'css/main.css'
```

Comment:
**[REQUIRED]**
Why are you ignoring your CSS file? :thinking:. Stylelint is a CSS linter and the purpose of using it is for checking the CSS file.  Linting is important to reduce errors and improve the overall quality of your code. Using the linter also gives a professional look in your repository. We need to remove this ignore rule and fix any errors if the stickler found any. You can also remove the ccslint since we are not using it.